web/modules/custom/custom_page/src/Controller/CustomPageController.php

```diff
@@ -66,14 +67,25 @@ public function add(RdfInterface $rdf_entity) {
   /**
    * Handles access to the custom page add form through collection pages.
    *
+   * Access is granted to moderators and group members that have the permission
+   * to create custom pages inside of their group, which in practice means this
+   * is granted to collection and solution facilitators.
```

Comment:
Here as well, collection members (not facilitators) might be able to add news. It depends on collection's settings.